### PR TITLE
Feat/top navbar(#61)

### LINKED ISSUE #61: [component] top navbar

### PR DESCRIPTION
**Pull request by aurelienroux**
RESOLVES #61 
top navbar with social links and search field

### CODE REVIEW

**Comment by f3ltron** [comment]
i check that tonight

File: components/navbar/navbar-top.vue [hunk 1]
```diff
@@ -1,11 +1,70 @@
...
 </script>
+
+<style>
+.icon {
+  left: -10px;
+  position: relative;
+}
+
+.icon svg {
+  height: 1.5rem !important;
```
Comment (fuentesloic) [inline components/navbar/navbar-top.vue]: is that a css-lib-overide??

Comment (aurelienroux) [reply components/navbar/navbar-top.vue]: yep, couldn't find a way to do this with Tailwind as the svg size is coming inline from FontAwesome

Comment (f3ltron) [reply components/navbar/navbar-top.vue]: if it's coming from fontawesome yeah you have no choice because it's about css selector length

**Pull request merged**